bundles/SystemInfoBundle/translations/admin.en.json

```diff
@@ -0,0 +1,6 @@
+{
+  "system_infos_and_tools": "System Info & Tools",
```

Comment:
Will face this and core translation when merging 11.x to fix conflicts anyways, but as reminder, we have to use YAML instead of JSON.

Reply:
good catch. fixed it 😉